data/courses.yaml

```diff
@@ -158,3 +158,10 @@
     but would now say that re-framing the goal as formalising something new in a format that is useful to you as a learner
     (e.g. as a worksheet/interactive part of a textbook) is a better goal pedagogically
     (and could still lead to, e.g., a mathlib contribution in the process even if that isn't the explicit goal).
+- name: Logic and Modelling
+  instructor: Rob Lewis, Jasmin Blanchette, Anne Baanen
```

Comment:
I list multiple instructors because Rob really is the one who designed the course, Jasmin took it over after Rob left and I took (most of) it over after Jasmin left. Or should we only list the current instructor?